doc/administrator/administrate.rst.tmpl

```diff
@@ -59,6 +59,10 @@ Some metadata items are used by the ``layers`` views (for editing) on the server
 * ``lastUpdateUserColumn``: Define a column which will be automatically filled with the user of the last edit.
 * ``readonlyAttributes``: Set attributes as read only.
 
+Some metadata items are used by the ``theme2fts` script:
+
+* ``searchAlias``: Comma separated list of search alias (keywords) to be added to the ``tsearch.ts``
```

Comment:
Is it clear what this tsearch.ts is, or what it stands for?
(It's not clear to me. I guess it means that a user search will match this element if it contains one of the search aliases).

Reply:
Hope it is clearer with:
```
* ``searchAlias``: Comma separated list of search aliases (keywords) to be added to the ``tsearch.ts``
  field for the considered tree item (theme, layer group or layer),
  that is to say that considered tree item will match when searching for the search aliases.
```